voyager_archive/static/css/style.css

```diff
@@ -27,4 +27,12 @@
   
   table.data-display tr:nth-child(even) {
     background-color: #f2f2f2;
+  }
+
+  .indicators {
+    color: green
+  }
+  
+  .subfield {
```

Comment:
Since this styles the subfield code aka the delimiter, and `delimiter` is the data used in the template, change this class to `delimiter`.